Makefile

```diff
@@ -12,18 +12,18 @@ test:
 	./gradlew connectedCheck
 
 remote-test:
-ifeq ($(BROWSER_STACK_USERNAME),)
-	@$(error BROWSER_STACK_USERNAME is not defined)
+ifeq ($(MAZE_DEVICE_FARM_USERNAME),)
+	@$(error MAZE_DEVICE_FARM_USERNAME is not defined)
 endif
-ifeq ($(BROWSER_STACK_ACCESS_KEY),)
-	@$(error BROWSER_STACK_ACCESS_KEY is not defined)
+ifeq ($(MAZE_DEVICE_FARM_ACCESS_KEY),)
+	@$(error MAZE_DEVICE_FARM_ACCESS_KEY is not defined)
 endif
 	@APP_LOCATION=/app/bugsnag-android-core/build/outputs/apk/androidTest/debug/bugsnag-android-core-debug-androidTest.apk \
 	 INSTRUMENTATION_DEVICES='["Google Nexus 5-4.4", "Google Pixel-7.1", "Google Pixel 3-9.0"]' \
 	 docker-compose up --build android-instrumentation-tests
 
 TEST_FIXTURE_NDK_VERSION ?= 16.1.4479499
-test-fixture:
+test-fixtures:
```

Comment:
The name should stay as-is IMO as only one APK is generated

Reply:
Ah, I had thought (without checking) that it was now building multiple, for the minimal test fixture.  There was also a driver to make it consistent with Cocoa so I don't keep getting it wrong when moving between repos, but I'm happy to change it back.